test/logging_test.dart

```diff
@@ -175,8 +175,8 @@ void main() {
     test('child zone', () {
       var root = Logger.root;
 
-      var recordingZone;
-      var records = new List<LogRecord>();
+      Zone recordingZone;
```

Comment:
I admit I'm confused.  Types are elided in some places and then added here...

Is this one right?

Reply:
Yup. `var` when inference work, be explicit otherwise. We cannot infer the type `Zone` here.

Reply:
Ugh, yeah.  That's what I thought.